src/Features/Core/Portable/SolutionCrawler/SolutionCrawlerRegistrationService.cs

```diff
@@ -134,22 +134,18 @@ public void Reanalyze(Workspace workspace, IIncrementalAnalyzer analyzer, IEnume
                 // no specific projects or documents provided
                 if (projectIds == null && documentIds == null)
                 {
-                    coordinator.Reanalyze(analyzer, workspace.CurrentSolution.Projects.SelectMany(p => p.DocumentIds).ToSet(), highPriority);
+                    coordinator.Reanalyze(analyzer, new ReanalyzeScope(workspace.CurrentSolution.Id), highPriority);
                     return;
                 }
 
                 // specific documents provided
                 if (projectIds == null)
```

Comment:
❔ Is this block needed anymore? It looks like the default value for `projectIds` in the `ReanalyzeScope` constructor is null, so it would work to just remove this check.